File: Entidad/Persona.cs
using System;
using System.IO;
using System.Security.AccessControl;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Entidad
{
    public class Persona
    {
        [Column(TypeName = "varchar(10)")]
        public string TipoDocumento {get;set;}

        [Key]
        [Column(TypeName = "varchar(15)")]
        public string Identificacion {get;set;}

        [Column(TypeName = "varchar(20)")]
        public string Nombre {get;set;}

        [Column(TypeName = "varchar(20)")]
        public string Direccion {get;set;}

        [Column(TypeName = "varchar(15)")]
        public string Telefono {get;set;}

        [Column(TypeName = "varchar(20)")]
        public string Pais{get;set;}

        [Column(TypeName = "varchar(20)")]
        public string Departamento {get;set;}

        [Column(TypeName = "varchar(20)")]
        public string Ciudad {get;set;}
    }
}

File: segundoparcial/ClientApp/src/app/gastosDiarios/models/persona.ts
import { Pago } from "./pago";

export class Persona {
    constructor(){
        this.pago = new Pago();
    }
    tipoDocumento: string;
    identificacion: string;
    nombre: string;
    direccion: string;
    telefono: string;
    pais: string;
    departamento: string;
    ciudad: string;
    pago: Pago;
    
}